src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -118,18 +123,31 @@ public CallbackService(final SecurityUtils securityUtils,
         HttpHeaders securityHeaders = securityUtils.authorizationHeaders();
 
         CallbackTelemetryThreadContext.setTelemetryContext(new CallbackTelemetryContext(callbackType));
+        List<String> ccdCallbackLogControl = applicationParams.getCcdCallbackLogControl();
+        boolean match = false;
+        String starStr = "*";//to match any call back
+        if (ccdCallbackLogControl.size() > 0 && (starStr.equals(ccdCallbackLogControl.get(0))
+            || ccdCallbackLogControl.stream().anyMatch(url::contains))) {
+            match = true;
+        }
         int httpStatus = 0;
         Instant startTime = Instant.now();
 
         try {
-            LOG.debug("Invoking callback {}", url);
             final HttpHeaders httpHeaders = new HttpHeaders();
             httpHeaders.add("Content-Type", "application/json");
             if (null != securityHeaders) {
                 securityHeaders.forEach((key, values) -> httpHeaders.put(key, values));
             }
             final HttpEntity requestEntity = new HttpEntity(callbackRequest, httpHeaders);
+            if (match) {
+                LOG.info("Invoking callback {} of CallbackType {} at {} with Request {}",
+                    url, callbackType, startTime, requestEntity);
+            }
             ResponseEntity<T> responseEntity = restTemplate.exchange(url, HttpMethod.POST, requestEntity, clazz);
+            if (match) {
+                LOG.info(" Received callback Response {}", responseEntity);
```

Comment:
I would log also the url to make it simpler to correlate the response to the callback. Otherwise it can be difficult when plenty of callbacks happen at the same time